editor/components/RelatedRecordTable.tsx

```diff
@@ -20,6 +20,16 @@ interface RelatedRecordTableProps<
    * will also be passed to the rowActionComponent, if present
    */
   mutation: string;
+  /**
+   * Function to generate the complete mutation variables payload
+   * If not provided, uses default behavior
+   */
+  getMutationVariables?: (
+    record: T,
+    column: ColDataCardDef<T>,
+    value: unknown,
+    defaultVariables: { id: number; updates: Record<string, unknown> }
+  ) => Record<string, unknown>;
```

Comment:
new prop that allows us to customize the update payload when editing a table a cell. this enables us to set the `_match_event_name` prop when a user manually types in a matching `person_id`.